### ISSUE TITLE
Python 2 & 3 compatibility

### ISSUE THREAD

**Issue by joshuago78** [gri-is/Python-CIDOC-ORM#2]
depends on #1

**Comment by joshuago78** [comment]
- Start with 2.7
- then 3.4 & 3.5
- we can address other versions later

**Comment by joshuago78** [comment]
I think I already covered this when I updated it to get automated testing working
